### PR TITLE
write unit test create book

### PR DESCRIPTION
**Pull request by ghost**
## Description
write unit test create book## Reference Requirements
- Task ID: TI-28

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-test/

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-nale-dn — COMMENTED** [review]
LGTM

File: tests/Browser/Pages/Backend/CreateBookTest.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    /**
+    * Override function set up database
+    *
+    * @return void
+    */
+    public function setUp()
+    {
+        parent::setUp();
+
+        factory(Category::class, 2)->create();
```
Comment (at-huydq) [inline tests/Browser/Pages/Backend/CreateBookTest.php]: test create thì ko cần khởi tạo trước

File: tests/Browser/Pages/Backend/CreateBookTest.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            $browser->visit('admin/books/create')
+                ->type('title', 'Title for book')
+                ->type('description', 'Lorem ABC')
+                ->type('number_of_page', '1000')
+                ->type('author', 'Cao Nguyen V.')
+                ->keys('#publishing_year', '11-22-2012')
+                ->type('quantity', '2');
+            $browser->press('Submit')
+                ->pause(1000)
+                ->assertSee('Successfully created book!');
```
Comment (at-huydq) [inline tests/Browser/Pages/Backend/CreateBookTest.php]: test thêm database có dữ liệu trùng vs data input ko 
https://laravel.com/docs/5.6/database-testing

File: tests/Browser/Pages/Backend/CreateBookTest.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Tests\Browser\Pages\Backend;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use App\Models\Category;
+
+class CreateBookTest extends DuskTestCase
```
Comment (at-huydq) [inline tests/Browser/Pages/Backend/CreateBookTest.php]: test thêm validate cho anh nữa nha.

**Review by ghost — COMMENTED** [review]
check my cmt

File: tests/Browser/Pages/Backend/CreateBookTest.php (hunk 1 above)
Comment (at-haytran) [reply tests/Browser/Pages/Backend/CreateBookTest.php]: mình ko khởi tạo Category nó sẽ không có dữ liệu để test anh ơi.

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**